packages/subgraph/src/mapping.ts

```diff
@@ -75,7 +74,7 @@ export function processPayload(
 
   while (reader.length() > 0) {
     let i = blockIdx.toString();
-    log.warning("New message block (num. {}) with remaining data: {}", [i, reader.data().toHexString()]);
+    log.info("New message block (num. {}) with remaining data: {}", [i, reader.data().toHexString()]);
```

Comment:
I usually place these in `warning` level since graph-node itself logs a TON of stuff in the `debug` and `info` levels, so having it in `warning` makes filtering for it a lot easier, but I know that it's not a best practice, just doing it for convenience